std/algorithm/iteration.d

```diff
@@ -4672,8 +4825,9 @@ Returns:
 
 See_Also:
  $(REF _splitter, std,regex) for a version that splits using a regular
-expression defined separator and
- $(REF _split, std,array) for a version that splits eagerly.
+expression defined separator, 
+ $(REF _split, std,array) for a version that splits eagerly and
+ $(LREF splitBy), useful for splitting without dropping any source range elements.
```

Comment:
`splitter` is going to get this feature in #7575, so you may want to reword this.